content/user/sideboxes/sidebox_list.md

```diff
@@ -23,6 +23,7 @@ This can be found in the setting <em>Admin > Configuration > Layout Settings</em
       <br><br>
       There are two more banner sideboxes, <strong>banner_box2</strong> and <strong>banner_box_all</strong>, which are configured similarly. <strong>Banner_box2</strong> is just a duplicate of the first, but <strong>banner_box_all</strong> is different in that it can display multiple banners randomly. 
    </div>
+	</div>
 <br clear="all">
 
 <ul>
```

Comment:
Get rid of this `<ul>`.  Look for others and get rid of them too - also `</ul>` and `<li>`, etc.

Reply:
You will want to use a plain text editor to find issues like this.   An HTML or Markdown editor might hide them from you.